src/NSubstitute.Analyzers.Shared/TinyJson/JsonParser.cs

```diff
@@ -302,7 +302,7 @@ private List<object> ParseArray()
             {
                 _json.Read(); // ditch opening brace
 
-                List<object> array = new List<object>();
+                var array = new List<object>();
```

Comment:
I like the `var` changes. I think changes like this may be better in a separate PR in future though, as it will make reviewing a lot easier. 😉 (yes, I am very selfish! 🤣)

Reply:
Agree, I was supposed to change it only in the TextParser file, but probably I've chosen the wrong option in Resharper ;]